core/kogitoinfra/keycloak_reconciler.go

```diff
@@ -15,9 +15,9 @@
 package kogitoinfra
 
 import (
-	keycloakv1alpha1 "github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
 	"github.com/kiegroup/kogito-operator/core/client/kubernetes"
 	"github.com/kiegroup/kogito-operator/core/infrastructure"
+	keycloakv1alpha1 "github.com/kiegroup/kogito-operator/core/infrastructure/keycloak/v1alpha1"
```

Comment:
maybe upgrade this api?

Reply:
`v1alpha1` is the latest keyclock-operator api.